ntbs-service/Pages/Notifications/Denotify.cshtml.cs

```diff
@@ -42,14 +42,19 @@ public async Task<IActionResult> OnGetAsync(int id)
                 return NotFound();
             }
 
+            await AuthorizeAndSetBannerAsync();
+            if (!HasEditPermission)
+            {
+                // Should not be able to get onto the denotify page for a notification user does not have edit access to
+                return Forbid();
```

Comment:
Seems to be a bit of a mix between `Forbid` and redirecting to Overivew, is there any specific reasoning behind which is used when?

Reply:
It seems that Forbid sends a 302 with a 'temporary redirect' to the below URL (based on our authentication scheme):
`/Account/AccessDenied?ReturnUrl=etc.`

https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.controllerbase.forbid?view=aspnetcore-3.0

I think proper handling of this should probably be handled as part of this ticket if it's using it quite extensively.
` return StatusCode((int)HttpStatusCode.Forbidden); ` might be a suitable stopgap which would let us consider it more thoroughly when we start to handle `NotFound` properly too.

Reply:
I agree this one should just redirect to overview as it's a get - forbids should only happen on post. I'll change it to status code as you suggest so that the redirect won't happen!